### PR TITLE
Speed optimizations

### PR DESCRIPTION
**Pull request by DBraun**
`_lfilter_unbatched` is now a little faster because I removed the nested `scan` function.

Inspired by [AudioTools](https://github.com/descriptinc/audiotools/blob/master/audiotools/core/loudness.py), I added `zeros=512` and `use_fir=False` keywords to `Meter` and `IIRfilter`. This is backwards compatible.

I removed `jax.jit` from `normalize.py` because it's better to let the user do that as late as possible.

I modified `test_loudness.py` to use both `use_fir=False` and `use_fir=True`. I chose an FIR size in order to pass each test.

```bash
python3 -m pytest -v test_loudness.py
```

Output:
```
test_loudness.py::test_integrated_loudness[False] PASSED                                                                                                                                                                           [  2%]
test_loudness.py::test_integrated_loudness[True] PASSED                                                                                                                                                                            [  4%]
test_loudness.py::test_batched_integrated_loudness[False] PASSED                                                                                                                                                                   [  6%]
test_loudness.py::test_batched_integrated_loudness[True] PASSED                                                                                                                                                                    [  8%]
test_loudness.py::test_peak_normalize[False] PASSED                                                                                                                                                                                [ 10%]
test_loudness.py::test_peak_normalize[True] PASSED                                                                                                                                                                                 [ 13%]
test_loudness.py::test_loudness_normalize[False] PASSED                                                                                                                                                                            [ 15%]
test_loudness.py::test_loudness_normalize[True] PASSED                                                                                                                                                                             [ 17%]
test_loudness.py::test_batched_loudness_normalize[False] PASSED                                                                                                                                                                    [ 19%]
test_loudness.py::test_batched_loudness_normalize[True] PASSED                                                                                                                                                                     [ 21%]
test_loudness.py::test_rel_gate_test[False] PASSED                                                                                                                                                                                 [ 23%]
test_loudness.py::test_rel_gate_test[True] PASSED                                                                                                                                                                                  [ 26%]
test_loudness.py::test_abs_gate_test[False] PASSED                                                                                                                                                                                 [ 28%]
test_loudness.py::test_abs_gate_test[True] PASSED                                                                                                                                                                                  [ 30%]
test_loudness.py::test_24LKFS_25Hz_2ch[False] PASSED                                                                                                                                                                               [ 32%]
test_loudness.py::test_24LKFS_25Hz_2ch[True] PASSED                                                                                                                                                                                [ 34%]
test_loudness.py::test_24LKFS_100Hz_2ch[False] PASSED                                                                                                                                                                              [ 36%]
test_loudness.py::test_24LKFS_100Hz_2ch[True] PASSED                                                                                                                                                                               [ 39%]
test_loudness.py::test_24LKFS_500Hz_2ch[False] PASSED                                                                                                                                                                              [ 41%]
test_loudness.py::test_24LKFS_500Hz_2ch[True] PASSED                                                                                                                                                                               [ 43%]
test_loudness.py::test_24LKFS_1000Hz_2ch[False] PASSED                                                                                                                                                                             [ 45%]
test_loudness.py::test_24LKFS_1000Hz_2ch[True] PASSED                                                                                                                                                                              [ 47%]
test_loudness.py::test_24LKFS_2000Hz_2ch[False] PASSED                                                                                                                                                                             [ 50%]
test_loudness.py::test_24LKFS_2000Hz_2ch[True] PASSED                                                                                                                                                                              [ 52%]
test_loudness.py::test_24LKFS_10000Hz_2ch[False] PASSED                                                                                                                                                                            [ 54%]
test_loudness.py::test_24LKFS_10000Hz_2ch[True] PASSED                                                                                                                                                                             [ 56%]
test_loudness.py::test_23LKFS_25Hz_2ch[False] PASSED                                                                                                                                                                               [ 58%]
test_loudness.py::test_23LKFS_25Hz_2ch[True] PASSED                                                                                                                                                                                [ 60%]
test_loudness.py::test_23LKFS_100Hz_2ch[False] PASSED                                                                                                                                                                              [ 63%]
test_loudness.py::test_23LKFS_100Hz_2ch[True] PASSED                                                                                                                                                                               [ 65%]
test_loudness.py::test_23LKFS_500Hz_2ch[False] PASSED                                                                                                                                                                              [ 67%]
test_loudness.py::test_23LKFS_500Hz_2ch[True] PASSED                                                                                                                                                                               [ 69%]
test_loudness.py::test_23LKFS_1000Hz_2ch[False] PASSED                                                                                                                                                                             [ 71%]
test_loudness.py::test_23LKFS_1000Hz_2ch[True] PASSED                                                                                                                                                                              [ 73%]
test_loudness.py::test_23LKFS_2000Hz_2ch[False] PASSED                                                                                                                                                                             [ 76%]
test_loudness.py::test_23LKFS_2000Hz_2ch[True] PASSED                                                                                                                                                                              [ 78%]
test_loudness.py::test_23LKFS_10000Hz_2ch[False] PASSED                                                                                                                                                                            [ 80%]
test_loudness.py::test_23LKFS_10000Hz_2ch[True] PASSED                                                                                                                                                                             [ 82%]
test_loudness.py::test_18LKFS_frequency_sweep[False] PASSED                                                                                                                                                                        [ 84%]
test_loudness.py::test_18LKFS_frequency_sweep[True] PASSED                                                                                                                                                                         [ 86%]
test_loudness.py::test_conf_stereo_vinL_R_23LKFS[False] PASSED                                                                                                                                                                     [ 89%]
test_loudness.py::test_conf_stereo_vinL_R_23LKFS[True] PASSED                                                                                                                                                                      [ 91%]
test_loudness.py::test_conf_monovoice_music_24LKFS[False] PASSED                                                                                                                                                                   [ 93%]
test_loudness.py::test_conf_monovoice_music_24LKFS[True] PASSED                                                                                                                                                                    [ 95%]
test_loudness.py::test_conf_monovoice_music_23LKFS[False] PASSED                                                                                                                                                                   [ 97%]
test_loudness.py::test_conf_monovoice_music_23LKFS[True] PASSED
```

GPU Benchmark test:
```bash
python3 -m pytest -s test_jit_benchmark.py
```
Output:
```
Num executions: 10
Mean execution time: 50.12 ms
Median execution time: 50.72 ms
Min execution time: 45.62 ms
Max execution time: 55.97 ms
Std execution time: 3.00 ms
```

### CODE REVIEW

**Comment by DBraun** [comment]
@boris-kuz What are your thoughts?

**Comment by boris-kuz** [comment]
Sorry, I completely missed this! This is great, thank you for the contribution!

**Comment by DBraun** [comment]
Thanks! Next question... would you be open to putting this on PyPI? I would like to have some of my other PyPI projects depend on it.
